ata_pipeline0/backfill.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime, timedelta
+from typing import List
+
+import click
```

Comment:
💯 This looks like a neat library!!

Reply:
It is! I figured I'd use this as an opportunity to try it out. The built in argparse is great but I like the lack of verbosity for this one; it's under the same umbrella project as Flask and Jinja